hazelcast-client-legacy/src/main/java/com/hazelcast/client/spi/impl/ClientMembershipListener.java

```diff
@@ -144,7 +144,7 @@ void initialMembers(ClientInitialMembershipEvent event) {
 
     private void memberRemoved(Member member) {
         members.remove(member);
-        applyMemberListChanges();
```

Comment:
Method was two lines. 
```
    private void applyMemberListChanges() {		
        updateMembersRef();		
        LOGGER.info(clusterService.membersString());		
  }
```
We don't need updateMembersRef anymore but rather updating the members in clusterService.handleMembershipEvent method. After that method become line, and I removed the method and put the one line here.